src/read/dwarf.rs

```diff
@@ -569,6 +569,20 @@ impl<R: Reader> Dwarf<R> {
         }
         err.description().into()
     }
+
+    /// Assuming `self` was loaded from a .dwo, take the appropriate
+    /// sections from `parent` (which contains the skeleton unit for this
+    /// dwo) and return a suitable instance of `Dwarf`.
+    pub fn make_dwo(mut self, parent: &Dwarf<R>) -> Dwarf<R> {
```

Comment:
```suggestion
    pub fn make_dwo(&mut self, parent: &Dwarf<R>) {
```

Also relax the requirement on `R` to be `R: Clone`.

Reply:
Done.

Reply:
How about the suggested change to be `&mut`? Since we're only modifying a few fields, that makes more sense to me, but I'm open to being persuaded.

Reply:
Ah I didn't actually look at the code :)

Sure, I don't care either way.